src/zabapgit_repo_impl.prog.abap

```diff
@@ -533,7 +535,16 @@ CLASS lcl_repo IMPLEMENTATION.
 
     lt_cache = mt_local.
     lt_tadir = lcl_tadir=>read( get_package( ) ).
-    LOOP AT lt_tadir ASSIGNING <ls_tadir>.
+
+    LOOP AT it_filter ASSIGNING <ls_filter>.
```

Comment:
Mmm, doesn't looping through `it_filter` have better performance than looping the entire `lt_tadir` assuming that the transport request have considerably less objects than the package?